packages/web/graphql/getQuest.ts

```diff
@@ -0,0 +1,68 @@
+import gql from 'fake-tag';
+import { Client } from 'urql';
+
+import {
+  GetQuestDocument,
+  GetQuestQuery,
+  GetQuestQueryVariables,
+  GetQuestWithCompletionsDocument,
+  GetQuestWithCompletionsQuery,
+  GetQuestWithCompletionsQueryVariables,
+} from './autogen/types';
+import { client as defaultClient } from './client';
+import {
+  PlayerFragment,
+  QuestFragment,
+  QuestWithCompletionFragment,
+} from './fragments';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+gql`
```

Comment:
We have sort of a trick here !

- Graphql codegen generates documents from all the queries it founds in the code (GetQuestsDocument)
- These documents are basically the same as what we have now, but with small differences (indentation/newlines)
- URQL caches requests based on a hash of the (query + variables)
- For some reasons, the hash generated from the query we write ourselves and the document auto-generated are different
- We need this cache, because NextJS pre-fetch the data serverside so that the client doesn't have to call graphql at load (see quests' page getStaticProps)

So we need to have this query written somewhere, but we don't need to have it stored in a variable ;)